picture.go

```diff
@@ -614,6 +614,52 @@ func (f *File) GetPictures(sheet, cell string) ([]Picture, error) {
 	return f.getPicture(row, col, drawingXML, drawingRelationships)
 }
 
+// GetAllPictures provides a function to get all pictures meta info and raw content
+// embed in spreadsheet by given worksheet. This function
+// returns the image contents as []byte data types. This function is
+// concurrency safe. For example:
+//
+//	f, err := excelize.OpenFile("Book1.xlsx")
+//	if err != nil {
+//	    fmt.Println(err)
+//	    return
+//	}
+//	defer func() {
+//	    if err := f.Close(); err != nil {
+//	        fmt.Println(err)
+//	    }
+//	}()
+//	picsMap, err := f.GetAllPictures("Sheet1")
+//	if err != nil {
+//		fmt.Println(err)
+//	}
+//	for cellName, pics := range picsMap {
+//		for _, pic := range pics {
+//			name := fmt.Sprintf("image%d%s", idx+1, pic.Extension)
+//			if err := os.WriteFile(name, pic.File, 0644); err != nil {
+//				fmt.Println(err)
+//			}
+//		}
+//	}
+func (f *File) GetAllPictures(sheet string) (map[string][]Picture, error) {
```

Comment:
Every exported function should have unit tests and the line coverage should approach 100%.